phpmyfaq/.htaccess

```diff
@@ -172,7 +172,7 @@ Header set Access-Control-Allow-Headers "Content-Type, Authorization"
     RewriteRule ^admin/api/(.*) admin/api/index.php [L,QSA]
 
     # Private APIs
-    RewriteRule ^api/(autocomplete|bookmark/([0-9]+)|user/data/update|user/password/update|user/request-removal|user/remove-twofactor|contact|voting|register|captcha|share|comment/create|faq/create|question/create) api/index.php [L,QSA]
+    RewriteRule ^api/(autocomplete|bookmark/remove/([0-9]+)|bookmark/add/([0-9]+)|user/data/update|user/password/update|user/request-removal|user/remove-twofactor|contact|voting|register|captcha|share|comment/create|faq/create|question/create) api/index.php [L,QSA]
```

Comment:
- Please don't add the ID in an API URL
- rename "remove" to "delete"
- rename "add" to "create"